packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -974,6 +973,7 @@ export class EthereumConnection implements AtomicSwapConnection {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
hmm, `_` or `_*` is the way to to express explicity that a variable is unused in TypeScript any many other languages. I think it would be better to adapt the linter https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md#argsignorepattern